templates/github/.github/workflows/update_ci.yml.j2

```diff
@@ -47,7 +47,7 @@ jobs:
       - name: Run update
         working-directory: {{ plugin_name }}
         run: |
-          .github/workflows/scripts/update_ci.sh
+          ../plugin_template/scripts/update_ci.sh {% if ci_update_docs -%} --docs {% endif -%} {{ plugin_name }} {{ noissue_marker | default('[noissue]') }}
```

Comment:
Yes, looks right to me. This now even covers the case where the update script on main would have been broken.
Can you add the old one to the deprecated_files please?